fibonacci.py

```diff
@@ -0,0 +1,48 @@
+# визначення числа в послідовності Фібоначчі, що відповідає прийнятому індексу:
+index = int(input("Enter an index of a number in Fibonacci sequence: "))
+
+# використовуючи генератори:
+def fibonacci_func(index):
+    num1 = 0
+    num2 = 1
+    if index == 0:
+        yield num1
+    elif index == 1:
```

Comment:
These two `elif` conditions are redundant. Also, you don't need to use so many `yield` inside of your generator, it is enough to use that one inside of `for in` block